nucliadb/nucliadb/train/generators/field_classifier.py

```diff
@@ -65,19 +65,21 @@ async def generate_field_classification_payloads(
     node: AbstractIndexNode,
     shard_replica_id: str,
 ) -> AsyncIterator[FieldClassificationBatch]:
-    labelset = f"/l/{trainset.filter.labels[0]}"
-
     # Query how many resources has each label
     request = StreamRequest()
     request.shard_id.id = shard_replica_id
-    request.filter.tags.append(labelset)
-    total = 0
 
+    labelset = None
```

Comment:
At this point should we only have 0 or 1 item in `trainset.filter.labels` ?

Reply:
Yes